src/config.cpp

```diff
@@ -41,7 +41,7 @@ enum preset_e : int {
 };
 
 enum rc_e : int {
-  constqp   = 0x0,  /**< Constant QP mode */
+  cqp       = 0x0,  /**< Constant QP mode */
```

Comment:
This will need to be updated in the docs as well, everywhere it changed. `./docs/source/about/advanced_usage.rst`